src/Layout/redist/MSBuildImports/Current/Microsoft.Common.CrossTargeting.targets/ImportAfter/Microsoft.TestPlatform.CrossTargeting.targets

```diff
@@ -93,14 +93,22 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
   ==================================================================================
-                                   _MTPBuild
+                                   _MTPTest
 
    Cross-targeting version of MTP.
 
   =================================================================================
   -->
-
-  <Target Name="_MTPBuild" DependsOnTargets="_ComputeTargetFrameworkItems">
+  <!-- This target is called by dotnet test for MTP for projects that are multi-targeted (using multiple target frameworks) -->
+  <Target Name="_MTPTest">
```

Comment:
NOTE: We should be careful to not regress `TestTfmsInParallel`. So maybe this outer `_MTPTest` target should also communicate back the `TestTfmsInParallel` value to the logger.